lib/ios-deploy.js

```diff
@@ -52,6 +52,10 @@ class IOSDeploy {
       return false;
     }
   }
+
```

Comment:
Alright. Updated so the only method is `isAppInstalled`, for both types of device.